### ISSUE TITLE
Check CBM 8032

### ISSUE THREAD

**Issue by polluks** [voidstar78/DestinyHunter#2]
for the sake of completeness...
http://www.zimmers.net/anonftp/pub/cbm/pet/utilities/cbm4032%20any%20hz.prg

**Comment by voidstar78** [comment]
An 8032 build is now available.  I examined the original PRG in VICE and thought about it for awhile.   While I can't easily make it a true 80 column game, I could easily just render in the left 40 column half of the screen.    However, the second thing I noticed is that the 8032 has different character codes for the KEYPAD (everything else seems the same, but it seems the 8032 updated the KEYPAD to essentially use the same codes as the Apple ][ - 0xB0 to 0xB9).    At the very beginning of the game, during the first ENTER - I use the detected keycode to auto-detect the difference between a 4032 and a 20XX/30XX.   If I instead have the user press NUMPAD 5, I could then also auto-detect the 80XX.  HOWEVER, I don't think I could guarantee whether that 80XX was actually in 80 or 40 column mode.  ANYWAY, for now I am giving up on trying to autodetect the 80-column mode, and am just doing a separate 8032 PRG build (dhunter8032.prg, I hope the name is not too long).    ALSO: the way my "graphics engine" works, doing a true 80-column game might not actually slow it down very much (the initial draw would be slower -- but after that, I only redraw exactly what was changed, I don't redraw the whole screen).  I'll ponder about that, maybe someday a true 80 column version could be made.       I hope this is what the Issue meant - in any case, a "nicer" 80-column 8032 compatible version is now available (for both the newer NUMPAD character codes, and at least being playable on the 80-column screen).
